frog/imports/client/Wiki/ModalDeletedPage.js

```diff
@@ -15,38 +14,34 @@ type ModalDeletedPagePropsT = {
  * or creating a new one.
  */
 export default ({
-  hideModal,
   onRestorePage,
   onCreateNewPage,
   pageTitle
 }: ModalDeletedPagePropsT) => {
   return (
-    <Modal
+    <AlertModal
```

Comment:
In the screenshot below, it is not so apparent to me that Home is the name of the page that you want to restore..  So maybe instead of having the page you want to restore as the title of the modal have in the body for this particular modal? 
<img width="1271" alt="Screen Shot 2019-07-16 at 3 32 01 PM" src="https://user-images.githubusercontent.com/16490844/61298967-8278a100-a7df-11e9-91a6-e913c91b0d77.png">

Reply:
Addressed change